src/components/marquee-wizard/marquee-wizard.component.ts

```diff
@@ -28,6 +28,9 @@ export class MarqueeWizardComponent extends WizardComponent implements OnDestroy
     /** If set to true the resizable splitter will be enabled and set to the default width **/
     @Input() resizable: boolean = false;
 
+    /** If set to true the nextinvalid will be enabled and current behavior will enable async validation. **/
```

Comment:
Consider using the same description you write in the docs for the property here. This is designed to be documentation for consumers without having to leave their editor, so it's good if the description here is on par with the docs